website/docs/r/container_node_pool.html.markdown

```diff
@@ -45,6 +45,10 @@ resource "google_container_cluster" "primary" {
       "https://www.googleapis.com/auth/logging.write",
       "https://www.googleapis.com/auth/monitoring",
     ]
+    guest_accelerator = [{
+      type="nvidia-tesla-k80"
+      count=1
+    }]
   }
```

Comment:
You'll need to add the details of this structure to the documentation below here.

Reply:
Done.